docs-examples/source/quickstart/Dockerfile-shell

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:latest
+FROM alpine:latest
```

Comment:
❓

Reply:
Oops, that was an experimental change I was testing a while back and shouldn't be included in this PR. This docker file isn't related to MongoDB (which requires glibc 6, not present in the Alpine distro).